src/network/networkd-mptcp.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 VMware, Inc. */
+#pragma once
+
+#include "conf-parser.h"
+#include "networkd-link.h"
+#include "networkd-network.h"
+#include "networkd-util.h"
+
+typedef struct MPTCP {
+        NetworkConfigSection *section;
+        Network *network;
+
+        uint16_t family;
```

Comment:
It is not necessary to use `uint16_t` for family. Please use `int` instead. Yes, I know that the netlink message use `_u16()` for family. But the value is correctly casted automatically.